javascript/javascript1/week3/lifeSpan.js

```diff
@@ -0,0 +1,32 @@
+const seriesDurations = [
+    {
+      title: "Game of thrones",
+      days: 3,
+      hours: 1,
+      minutes: 0,
+    },
+    {
+      title: "Sopranos",
+      days: 3,
+      hours: 14,
+      minutes: 0,
+    },
+    {
+      title: "The Wire",
+      days: 2,
+      hours: 12,
+      minutes: 0,
+    },
+  ];
+  const lifeDurationInMinutes = 80 * 365 * 24 * 60 ;
+  let sumSeriesTIme = 0;
+  function logOutSeriesText(seriesDurations) {
+    for(let i =0;i<seriesDurations.length;i++){
+        seriesTimeInMinutes = (seriesDurations[i].days  * 24 * 60) + (seriesDurations[i].hours * 60) + (seriesDurations[i].minutes);
```

Comment:
`seriesTimeInMinutes` need a `const` keyword.

Reply:
If you want to make this line shorter, you can use something called "[destructuring](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#object_destructuring)".
It might be too soon to learn it you can use this syntax:
```javascript
const { days, hours, minutes } = seriesDurations[i];  // This is the same as doing: "const days = seriesDurations[i].days" etc.

const seriesTimeInMinutes = (days  * 24 * 60) + (hours * 60) + (minutes);
```

Reply:
oh i forgot it, i will put the const keyword, and i will learn about destructuring and replace te code,
thanks for the concept.

Reply:
but here if it was just only days, hour and minutes then we can do like that, but iam convertiig their values in minutes individually , fx: day * 24 * 60 , for hours : hours *60 
so if i write const { days, hours, minutes } = seriesDurations[i]; 
it will just do seriesDurations[i].days , seriesDurations[i].hours and minutes , but not convert into minues 👍 so how can we convert in minutes on destructuring way? i got confused ::

Reply:
Yeah, I get why you're confused.
```js
const { minutes, days, hours } = seriesDurations[i]; 
// now "minutes" is the same as writing "seriesDurations[i].minutes", 
// because you've dragged out the property.
// So:
const seriesTimeInMinutes = (days  * 24 * 60) + (hours * 60) + (minutes); 
// is the same as writing
const seriesTimeInMinutes = (seriesDurations[i].days  * 24 * 60) + (seriesDurations[i].hours * 60) + (seriesDurations[i].minutes);
```